### PR TITLE
Allow Admin to limit what nodes can be installed

### PR DESCRIPTION
**Pull request by hardillb**
Part of https://github.com/flowforge/flowforge/issues/216

### CODE REVIEW

File: forge/routes/api/template.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ module.exports = async function (app) {
      */
     app.get('/:templateId', async (request, reply) => {
         const template = await app.db.models.ProjectTemplate.byId(request.params.templateId)
+        console.log(template)
```
Comment (knolleary) [inline forge/routes/api/template.js:31]:
```suggestion
```

**Review by knolleary — APPROVED** [review]

File: forge/routes/api/template.js (hunk 1 above)
Comment (hardillb) [reply forge/routes/api/template.js:31]: Doh, missed one

**Comment by knolleary** [comment]
urg. I spot the console.log, I comment on the console.log, I propose the removal of the console.log..... I click merge anyway.

File: forge/routes/api/template.js (hunk 1 above)
Comment (hardillb) [reply forge/routes/api/template.js:31]: Err, you've merged this before this was fixed...

Comment (knolleary) [reply forge/routes/api/template.js:31]: I am as disappointed about it as you are.

**Pull request merged**